src/infrastructure/ui/shared/component/inputs/generic-input-with-search-icon/generic-input-with-search-icon.tsx

```diff
@@ -30,7 +33,9 @@ const GenericInputWithSearchIcon = ({
 
                 <TouchableOpacity
                     style={GenericInputWithSearchIconStyle.containerSearchIcon}
-                    onPress={onPressSearchIcon}>
+                    onPress={() => {
+                        onPressSearchIcon();
+                    }}>
```

Comment:
Pour que ce soit plus propre, je preferais l'ancienne méthode
```suggestion
                                        onPress={onPressSearchIcon}>
```